README.md

```diff
@@ -1,73 +1,100 @@
 
-A description of all input variables (i.e. variables that are defined in
-`defaults/main.yml`) for the role should go here as these form an API of the
-role.
+Unconfigured, this role will just report system status as described in the
+following section.
 
-Variables that are not intended as input, like variables defined in
-`vars/main.yml`, variables that are read from other roles and/or the global
-scope (ie. hostvars, group vars, etc.) can be also mentioned here but keep in
-mind that as these are probably not part of the role API they may change during
-the lifetime.
+ * `crypto_policies_policy`
 
-Example of setting the variables:
+Use this variable to specify the desired crypto policy on the target system,
+which can be either the base policy or a base policy with policy submodule(s)
+as accepted by the `update-crypto-policies` tool.For example `FUTURE` or
+`DEFAULT:NO-SHA1:GOST`. The specified base policy and policy submodule(s)
+must be available on the target system.
 
-```yaml
-template_foo: "oof"
-template_bar: "baz"
-```
+The default value is `false` meaning the configuration is not changed and
+the role will just collect the facts below.
+
+ * `crypto_policies_reload`
+
+By default (`true`), updating crypto policies forces reload of some of
+the daemons affected by crypto policies in the system. Setting `false`
+prevents this behavior and is helpful if the role is executed during system
+enrollment or some other follow-up tasks is expected to do it later.
+
+ * `crypto_policies_reboot`
+
+Crypto policies can not know all the custom applications using crypto
+libraries that are affected by change of crypto policies so it is recommended
+to reboot after changing crypto policies to make sure all of the services
+and applications will read the new configuration files. By default (`false`),
+if reboot is required, this role will set `crypto_policies_reboot_required`
+variable as described below and it is up to the user of the role to reboot
+system afterwards, for example after applying some other changes that might
+need reboot. If there are no other tasks that require playbook, you can set
+this value to `true` and this role will handle the reboot for you, when needed.
 
 ### Variables Exported by the Role
 
-This section is optional.  Some roles may export variables for playbooks to
-use later.  These are analogous to "return values" in Ansible modules.  For
-example, if a role performs some action that will require a system reboot, but
-the user wants to defer the reboot, the role might set a variable like
-`template_reboot_needed: true` that the playbook can use to reboot at a more
-convenient time.
+* `crypto_policies_active`
 
-Example:
+This fact contains the currently active policy name in the format as accepted
+by `crypto_policies_policy` variable above.
 
-`template_reboot_needed` - default `false` - if `true`, this means
-a reboot is needed to apply the changes made by the role
+* `crypto_policies_available_policies`
```

Comment:
Have you considered calling it just `crypto_policies_available`?
On one hand, I understand that `crypto_policies_` is just a namespace prefix and the description of the variable comes after.
On the other hand, `crypto_policies_available_policies` looks silly as a needlessly repetititive repetitition.

Reply:
The problem is that the policy name accepted as input to `crypto_policies_policy` will take not only the "bare" policies from `crypto_policies_available_policies`, but also combination of them with 1 or more `crypto_policies_available_modules`.

That is why we wanted to keep these two `available` lists similar in naming to make sure they are related to each other, rather than having one just `available` and the other about modules. Not sure if it is helpful.